donate/templates/payment/card_master.html

```diff
@@ -67,15 +63,19 @@ <h1 class="heading heading--primary heading--bottom-margin">{% trans "Your Detai
                                     {% endfor %}
                                     {% trans "Please correct these errors and try again." %}
                                 </div>
+                            {% else %}
```

Comment:
What this section is now doing is saying: 
"If there are any errors on the "your details" form, display this translated message instead of refreshing the page."

However, if the form is completed and submitted, but there is an error on the braintree side or a network error, the page will refresh and will render the appropriate error